src/gui/qgsconfigureshortcutsdialog.cpp

```diff
@@ -280,20 +280,110 @@ void QgsConfigureShortcutsDialog::loadShortcuts()
   QString actionSettingKey;
 
   QDomElement child = root.firstChildElement();
+  bool reassignAll = false;
+  bool skipAll = false;
```

Comment:
since these are mutually exclusive I'd lean towards a private enum, eg:

    enum class ActionOnExisting
    {
        Ask,
        ReassignAll,
        SkipAll
    };
    ActionOnExisting onExisting = ActionOnExisting::Ask;

Reply:
Good idea.